src/components/Header/Header.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import * as React from 'react';
+
+import classNames from 'classnames';
+import Link from 'next/link';
+import { useSelectedLayoutSegment } from 'next/navigation';
+import { signOut } from 'next-auth/react';
+
+const Header = () => {
+  const segment = useSelectedLayoutSegment();
+  return (
+    <header className="app-header">
+      <nav className="app-header__navigation">
+        <div className="app-header__menu">
+          <Link
+            href="/"
+            className={classNames('app-header__brand', {
```

Comment:
`-brand is usually for the logo Since it is a menu item it could be:

```suggestion
            className={classNames('app-header__menu-item', {
```

or simply:

```suggestion
            className={classNames('app-header__link', {
```

I missed that in the first review 🙇

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/80/commits/702ee846f6f726d9978aaa81f06aa9da4a3aea7c